system-design/app-interoperability/observability.md

```diff
@@ -54,13 +57,13 @@ The device owner MUST NOT use the sidecar deployment model at this time since th
 
 > **Action:** Some more research needs to be done here. If there is a way to do this dynamically without requiring the application developer to include special attributes on their pods then it may be allowed.
 
-The device owner MUST NOT pre-configure any exporters that would make any observability data available outside the device/cluster.
+The device owner MUST NOT pre-configure exporters to send observability data from the device because the end user must control what observability data is exported.
```

Comment:
would like to rephrase this one. 
" The device owner MUST provide the end user the ability to configure the observability data endpoint."

Reply:
That's not the point of this line. Line 167 has the statement you're suggesting. 

The point of line 60 is to indicate device owners should not automatically export any observability data and the end user must configure any exports they want to use so the customer is aware of what is exported and has control over what gets exported.